.github/workflows/remoteconfig.yml

```diff
@@ -53,13 +51,14 @@ jobs:
     strategy:
       matrix:
         target: [ios, tvos, macos, watchos]
+        podspec: [FirebaseRemoteConfig.podspec, FirebaseRemoteConfigSwift.podspec --skip-tests]
```

Comment:
The FirebaseRemoteConfigSwift.podspec require setup for the console access are handled by `build.sh` above.